app/components/PageMain/PageMain.js

```diff
@@ -1,16 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import SettingsPanel from '../SettingsPanel/SettingsPanel';
 
-export default function PageMain({ setBoardColor, }) {
-  const panelProps = { setBoardColor, };
+
+export default function PageMain({ children, }) {
```

Comment:
MainPage is better English

Reply:
Although I do agree about the sound of it, I wanted to create a separation of a page's header and a page's main section. Which in this case does not necessarily reflect properly in the phrasing 'main page'.

Perhaps I should rename both to *Purpose*Section..?

Reply:
FooSection sounds good. I usually set a naming convention, where each file which is a subset of one of the main entity types has it as a suffix in the name. So given these base entities: "Page", "Route", "View", "Widget", "Model", "Store", "Foo", files would be named "HomePage", "RegistrationRoute", "AddCommentView", "BuySellWidget", "UserModel", "OrdersStore".
It is up to you to define which entities your app has and decide on a naming convention which would make it easier for you to reason about, navigate and make changes in your app.

Reply:
done